tools/common/Assembly.cs

```diff
@@ -354,6 +354,12 @@ public void ComputeLinkerFlags ()
 					case "libdyld":
 					case "libsystem_kernel":
 						break;
+					case "sqlite3":
+						if (LinkWith == null)
+							LinkWith = new List<string> ();
+						LinkWith.Add ("-lsqlite3");
+						Driver.Log (3, "Linking with {0} because it's referenced by a module reference in {1}", file, FileName);
+						break;
```

Comment:
minor:it could have been merged with the following `"libsqlite3`" case, even if that one could eventually be re-used differently.